install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -843,7 +843,7 @@ spec:
               - key: ca.crt
                 path: common-etcd-client-ca.crt
           {{- else }}
-          - configMap:
+          - {{ if .Values.tls.caBundle.useSecret }}secret{{ else }}configMap{{ end }}:
```

Comment:
So there are multiple uses of `.Values.tls.caBundle.enabled` in our Helm charts. I think you will have to change all of them to use this pattern.